Remove old hexadecimal and octal syntax. (#1485)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

TERMINATE TRANSACTION[S] transaction-id[,...]
----

a|
label:functionality[]
label:removed[]
[source, cypher, role="noheader"]
----
SHOW TRANSACTIONS YIELD allocatedBytes
----
a|

> This column has been removed as it wasn't tracked and always returned 0.

Reviewer: ```suggestion
The `allocatedBytes` output has been removed, because it was never tracked and thus was always 0.
```

> 

Reviewer 2: Did you mean to include this? Doesn't seem related to Octal or Hex

Author: Actually I just moved it up two steps to be together with the other SHOW TRANSACTION entries, but looks like Github is not presenting it very well

Reviewer 2: Oh yes my bad :)

----

a|
label:syntax[]
label:removed[]
[source, cypher, role="noheader"]
----

> 0...

Reviewer 2: Same question here. Suspicious!
